source/stylesheets/refills/_ribbon.scss

```diff
@@ -0,0 +1,33 @@
+.ribbon-box {
+  @include size(100px);
+  position: relative;
+  border: 1px solid #bbb;
+}
+
+.ribbon-wrapper {
+  $base-accent-color: #477DCA !default;
+  $ribbon-background: $base-accent-color;
+  $ribbon-size: 85px;
+
+  @include size($ribbon-size);
+  overflow: hidden;
+  position: absolute;
+  right: -1px;
+  top: -1px;
+
+  .ribbon {
+    @include transform(rotate(45deg));
+    background-color: $ribbon-background;
+    box-shadow: 0px 0px 3px rgba(0,0,0,0.3);
```

Comment:
Commas in function arguments should be followed by a single space<br>`0px` should be written without units as `0`